brambling/templates/brambling/forms/signup.html

```diff
@@ -23,7 +23,7 @@
 <div class="row">
 	<div class="form-group col-xs-12">
 		<a href="#extraName" data-toggle="collapse" class="{% if masthead %}masthead-link {% endif %}pull-right more-name-options"><small><i class="fa fa-plus"></i> More Name Options</small></a>
-		{% formrow form.email %}
+		{% formrow form.email with value=request.GET.email %}
```

Comment:
Why handle this here instead of adding request.GET.email to the forms initial data in python? I would rather see it that way.